docs/sources/reference/stdlib/base64_decode.md

```diff
@@ -0,0 +1,19 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/stdlib/base64_decode/
+description: Learn about base64_decode
+title: base64_decode
+---
+
+# base64_decode
+
+The `base64_decode` function decodes a RFC4648-compliant Base64-encoded string 
+into the original string. 
+
+`base64_decode` fails if the provided string argument contains invalid Base64 data. 
+
+## Examples
+
+```
+> base64_decode("Zm9vYmFy")
+foobar
```

Comment:
Can we use some other more generic example. We should try to avoid using `foo`, `bar`, and `bas` as placeholders or generic terms. Refer to: https://developers.google.com/style/word-list#foo which we use as a guideline.

Reply:
done in HEAD! :D